shoop/front/apps/auth/forms.py

```diff
@@ -103,5 +103,5 @@ def process_user(self, user_to_recover):
         subject = loader.render_to_string(self.subject_template_name, context)
         subject = ''.join(subject.splitlines())  # Email subject *must not* contain newlines
         email = loader.render_to_string(self.email_template_name, context, request=self.request)
-        send_mail(subject, email, self.from_email, [user_to_recover.email])
+        send_mail(subject, email, self.from_email, [getattr(user_to_recover, 'email', '')])
```

Comment:
Here we don't want to send the mail at all if the user model does not have email available.